v2/pink-sb/src/lib/expandable-table/cell/Cell.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+    import type { RootProp } from '../index.js';
+    import Icon from '$lib/Icon.svelte';
+    import { IconChevronDown } from '@appwrite.io/pink-icons-svelte';
+    import Button from '$lib/button/Button.svelte';
+    import { getContext } from 'svelte';
+
+    export let root: RootProp;
+    export let column: string;
+    export let expandable: boolean = true;
+    const rowId = getContext<string>('rowId');
+
+    $: columnIndex = root.columns.findIndex((c) => c.id === column);
+    $: justify = root.alignment(root.columns[columnIndex]?.align);
```

Comment:
we don't need the `root.alignment` method. see - [v2/pink-sb/src/lib/table/cell/Base.svelte](https://github.com/appwrite/pink/blob/next/v2/pink-sb/src/lib/table/cell/Base.svelte)